DEVELOPING.md

```diff
@@ -1,32 +1,34 @@
 # Developing
 
+## Local Development
+
```

Comment:
This could be :

- fork `git@github.com:adamghill/django-unicorn.git`
- `git clone` your repository version
- `cd django-unicorn`
- ......
cloning the project directly will cause git permission issue when it's time to push new changes

Reply:
Added, thanks for the suggestion. 👍